apps/emqx_machine/src/user_default.erl

```diff
@@ -0,0 +1,28 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2021-2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+-module(user_default).
+
+-include_lib("emqx/include/emqx.hrl").
+-include_lib("emqx/include/logger.hrl").
+-include_lib("emqx/include/emqx_mqtt.hrl").
+-include_lib("emqx_conf/include/emqx_conf.hrl").
+-include_lib("emqx_dashboard/include/emqx_dashboard.hrl").
```

Comment:
why the includes ?

Reply:
For importing record definition to shell.
```erlang
EMQ X Community Edition 5.0.0-beta.3-e418e380 is running now!
Restricted Eshell V12.0.2  (abort with ^G)
5.0.0-beta.3-e418e380(emqx@127.0.0.1)1> #mqtt_packet_connect{}.
#mqtt_packet_connect{proto_name = <<"MQTT">>,proto_ver = 4,
                     is_bridge = false,clean_start = true,will_flag = false,
                     will_qos = 0,will_retain = false,keepalive = 0,
                     properties = #{},clientid = <<>>,will_props = #{},
                     will_topic = undefined,will_payload = undefined,
                     username = undefined,password = undefined}
```

Reply:
Please add a comment to explain it